config/unicorn.rb

```diff
@@ -2,4 +2,16 @@
 
 GovukUnicorn.configure(self)
 
+# Preload the entire app. By preloading an application you can save some
+# RAM resources as well as speed up server boot times.
+preload_app true
+
+after_fork do
+  # Throttles the master from starting new workers too quickly.
+  # Creates a more gradual roll out of workers so that users are
+  # less likely to have their request handled by a worker that has
+  # just booted, and may respond to requests slower.
+  sleep 5 unless ENV["RACK_ENV"] == "development"
```

Comment:
This definitely seems like a good improvement, my only concern is that unicornherder has a fixed amount of time it waits before it kills off the old processes.  So if we add more workers (or increase this timeout), we run the risk of not having all the new workers up before the old ones go away.

Not a problem now, but something to keep an eye on in the future.

Reply:
Yeah I found 10 seconds was too long, because very shortly after the final new worker had started all of the previous workers quit.